### PR TITLE
Allow wildcard matching

### PR DESCRIPTION
**Pull request by tastycode**

### CODE REVIEW

File: Contents/Sketch/lib/TextLayerSync.js [hunk 1]
```diff
@@ -29,6 +34,19 @@ var TextLayerSync = function(layer)
...
+    function wildcardMatch(key, store, language) {
+      var rex = new RegExp(key.replace('*', '.*'), 'i');
+      var candidateKeys = Object.keys(store);
+      var matchedKeys = candidateKeys.filter(function(key) {
+        return rex.test(key);
+      });
+      if (matchedKeys.length != 0) {
+        var randomMatchedKey = randomArrayItem(matchedKeys)
+        return store[randomMatchedKey][language];
+      }
```
Comment (tastycode) [inline Contents/Sketch/lib/TextLayerSync.js]: I can totally change this to be explicit with a `return null` at the end. 

> if this function always returns something

 A function with no return statement implicitly returns null. There isn't a difference between `function() {}` and `function() { return null }`. From the outside, they behave exactly the same . So adding `return null` would only be for explicit readability, which is reasonably salient.

File: README.md [hunk 2]
```diff
@@ -40,6 +40,12 @@ A synchronized element's content can also be based on concatenating multiple `<k
 
 To add new lines between items, use `'\n'` for example `sync:<key1> + '\n' + <key2>`.
 
+## Content wildcards
+
+A synchronized element's content can be chosen by pattern. For example
```
Comment (tastycode) [inline README.md]: Yeah... I don't totally understand how this plugin works with symbol layers, do you think there is functionality that should be added to work with symbols?

Comment (dbrody) [inline README.md]: OK, sounds good. It seems like the main feature here is for wildcard uses. I think this is a great first go at it.

**Comment by dbrody** [comment]
OK, this will be merged. I am following it up with another merge for small modifications plus a version update to v0.3.3.

Thank you for the contribution @tastycode! Keep them coming. 👍

**Pull request merged**